Phindr3D-Python/src/Data/Metadata.py

```diff
@@ -105,7 +147,10 @@ def metadataFileExists(self, omf):
     pass
 
 
-
+test = Metadata()
+test.SetMetadataFilename('C:/Users/fyi/Desktop/metaout_metadatafile.txt')
+test.loadMetadataFile()
+print(test.images[4.0].layers[11].channels[2].channelpath)
```

Comment:
Include a value or file to test against. What is expected, and how does it compare? Print "Load metadata file test: " and "Success" or "Failure".